stacks/ucan-invocation-stack.js

```diff
@@ -0,0 +1,50 @@
+import {
+  Bucket,
+  Function,
+  use
+} from '@serverless-stack/resources'
+
+import { BusStack } from './bus-stack.js'
+import { getConfig, setupSentry } from './config.js'
+
+/**
+ * @param {import('@serverless-stack/resources').StackContext} properties
+ */
+export function UcanInvocationStack({ stack, app }) {
```

Comment:
minor: perhaps this is the ucan-archive-stack ?
```suggestion
export function UcanArchiveStack({ stack, app }) {
```

Reply:
it is not archive in reality. We do other kind of post processing on it. I like `invocation` because of being a spec term https://github.com/ucan-wg/spec#29-invocation that is what we are doing.